src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -48,44 +52,40 @@ export interface Props {
    */
   id?: string;
   /**
-   * Passed down to initially set the activeIndex state
-   */
-  activeIndex?: number;
-  /**
-   * The array of items to be passed into the component. The array must take on the specified shape
-   * TODO: improve `any` type
+   * Overflow variants
+   * - **inverted** changes the overflow shadow to the inverted color
    */
-  items?: Array<any>;
+  overflow?: 'inverted';
   /**
    * Indicates that field is required for form to be successfully submitted
    */
   required?: boolean;
+  /**
+   * Right slot: content that will be positioned to the right of title
+   */
+  right?: ReactNode;
```

Comment:
I'm wondering if there is a more specific prop name we could use here, like `rightElement` or something of that nature.

Reply:
Good catch. The `<PageHeader>` component uses the same prop and calls it `titleAfter` so I changed this to match.

Reply:
Awesome, thanks!